tests/assets/classification_dataset_class_incremental/3/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Is this file intended?

Reply:
Yes, from https://github.com/openvinotoolkit/training_extensions/pull/2606 to simulate a label without annotation.